projects/documents.py

```diff
@@ -0,0 +1,21 @@
+from django_elasticsearch_dsl import Document, Index, fields
```

Comment:
You don't use `fields` here, so you can remove it from the imports.

Why use both a global `project_index` and a nested `Index` class? One is sufficient since the `Index` class inside the Document sets the index name.